account_payment_order/views/account_payment_line.xml

```diff
@@ -88,6 +88,7 @@
                 <field name="amount_currency" string="Amount" />
                 <field name="currency_id" invisible="1" />
                 <field name="name" optional="show" />
+                <field name="company_currency_id" invisible="1" />
```

Comment:
I think that in the tree view, it should use the `column_invisible` attribute instead of `invisible`.
![image](https://github.com/user-attachments/assets/2c00c952-25a3-47b1-a9b9-4edd2dc53b00)

Reply:
Agh, true. @micheledic can you please do a PR fixing it